src/compile/data/nullfilter.ts

```diff
@@ -60,7 +57,7 @@ export const nullFilter: DataComponentCompiler<Dict<FieldDef>> = {
       const childDataComponent = child.component.data;
       if (model.compatibleSource(child) && !differ<FieldDef>(childDataComponent.nullFilter, nullFilterComponent)) {
         extend(nullFilterComponent, childDataComponent.nullFilter);
-        delete childDataComponent.nullFilter;
+        childDataComponent.nullFilter = {};
```

Comment:
I was expecting nullFilter to be always defined. If I don't do this, `Object.keys` breaks.